### PR TITLE
267 BRES to NUTS 2, 3 and LEP

### LINKED ISSUE #267: BRES data to NUTS3 and LEPs

### PR DESCRIPTION
**Pull request by dede95**
…d data files not needed

Closes #267

### CODE REVIEW

File: ds/data/processed/industry/economic_complexity_index.nuts2.csv [hunk 1]
```diff
@@ -1,4 +1,4 @@
-year,nuts_id,nuts_year_spec,economic_complexity_index
+year,lep_id,lep_year_spec,economic_complexity_index
```
Comment (georgerichardson) [inline ds/data/processed/industry/economic_complexity_index.nuts2.csv]: Looks like this nuts2 file has lep headings

File: ds/data/processed/industry/employment_culture_entertainment_recreation.nuts2.csv [hunk 2]
```diff
@@ -1,4 +1,4 @@
-year,nuts_id,nuts_year_spec,employment_culture_entertainment_recreation
+year,lep_id,lep_year_spec,employment_culture_entertainment_recreation
```
Comment (georgerichardson) [inline ds/data/processed/industry/employment_culture_entertainment_recreation.nuts2.csv]: this one also has lep headings

File: ds/data/processed/industry/economic_complexity_index.nuts3.csv [hunk 3]
```diff
@@ -0,0 +1,693 @@
+year,lep_id,lep_year_spec,economic_complexity_index
```
Comment (georgerichardson) [inline ds/data/processed/industry/economic_complexity_index.nuts3.csv]: lep headers

File: ds/data/processed/industry/employment_culture_entertainment_recreation.lep.csv [hunk 4]
```diff
@@ -0,0 +1,153 @@
+year,lep_id,lep_year_spec,employment_culture_entertainment_recreation
+2016,E37000001,2017,20930
+2016,E37000002,2017,9985
+2016,E37000003,2017,18320
+2016,E37000004,2017,39975
+2016,E37000005,2017,10435
```
Comment (georgerichardson) [inline ds/data/processed/industry/employment_culture_entertainment_recreation.lep.csv:6]: i think that data from any years before 2017 should have a `lep_year_spec` of 2014

File: ds/data/processed/industry/employment_culture_entertainment_recreation.nuts3.csv [hunk 5]
```diff
@@ -0,0 +1,673 @@
+year,lep_id,lep_year_spec,employment_culture_entertainment_recreation
```
Comment (georgerichardson) [inline ds/data/processed/industry/employment_culture_entertainment_recreation.nuts3.csv]: lep headings

File: ds/data/processed/sample.csv [hunk 6]
```diff
@@ -0,0 +1,51 @@
+postcode,Average download speed (Mbit/s),latitude,longitude,year
```
Comment (georgerichardson) [inline ds/data/processed/sample.csv]: i think this temporary file can be deleted?

File: ds/notebooks/dev/Untitled.ipynb [hunk 7]
```diff
@@ -0,0 +1,230 @@
+{
```
Comment (georgerichardson) [inline ds/notebooks/dev/Untitled.ipynb]: this notebook doesn't have a title

File: ds/model_config.yaml [hunk 8]
```diff
@@ -4,7 +4,9 @@ data:
 
   nomis:
     years: [2019,2018,2017,2016]
```
Comment (georgerichardson) [inline ds/model_config.yaml]:
```suggestion
    years: [2019, 2018, 2017, 2016]
```

File: ds/model_config.yaml [hunk 9]
```diff
@@ -4,7 +4,9 @@ data:
...
-    geography: TYPE450
+    geography: TYPE459
```
Comment (georgerichardson) [inline ds/model_config.yaml:7]: to create this indicator does the user need to manually switch between geography types?

File: ds/data/processed/industry/economic_complexity_index.lep.csv [hunk 10]
```diff
@@ -0,0 +1,153 @@
...
+2017,E37000034,2017,-0.1427
+2017,E37000035,2017,0.2845
+2017,E37000036,2017,-0.1112
+2017,E37000037,2017,0.1882
+2017,E37000038,2017,-0.0835
+2017,E37000039,2017,-0.0794
+2017,E37000040,2017,-0.1299
+2017,E37000041,2017,0.1063
+2017,E37000042,2017,-0.0189
+2016,E37000001,2017,-0.1332
```
Comment (georgerichardson) [inline ds/data/processed/industry/economic_complexity_index.lep.csv:116]: 2016 data should have 2014 lep_year_spec (is possible?)

**Review by georgerichardson — COMMENTED** [review]
I've made a general comment about the use of LEP 2017 where LEP 2014 might be needed, but I'm not sure if this is limited by the API? Also, all of the NUTS data seems to have nuts_year_spec of 2013, but again maybe that's the nature of the API/data.

File: ds/model_config.yaml (hunk 9 above)
Comment (dede95) [reply ds/model_config.yaml:7]: Yes, I had to manually switch when working on the different types.

File: ds/data/processed/industry/employment_culture_entertainment_recreation.lep.csv (hunk 4 above)
Comment (dede95) [reply ds/data/processed/industry/employment_culture_entertainment_recreation.lep.csv:6]: Based on the codelist for the API, I think only the LEP regions available from 2017. I'll double check again

**Comment by mindrones** [comment]
The schema has changed after #365, please make sure to update this PR before merging.

**Review by mindrones — APPROVED** [review]
Good to merge for my part 👍

**Pull request merged**